Sources/TokamakCore/Fiber/Fiber+CustomDebugStringConvertible.swift

```diff
@@ -22,7 +22,7 @@ extension FiberReconciler.Fiber: CustomDebugStringConvertible {
     {
       return "Text(\"\(text.storage.rawText)\")"
     }
-    return typeInfo?.name ?? "Unknown"
+    return "\(typeInfo?.name ?? "Unknown") \(ObjectIdentifier(self).debugDescription.split(separator: "(")[1].split(separator: ")")[0])"
```

Comment:
Can we guarantee that `ObjectIdentifier(self).debugDescription.split(separator: "(")` will always return at least two elements? Maybe we need to guard for that?

Reply:
I just removed this from the description, since I was just using it to debug something. If you think I should keep the memory address though, using `unsafeBitCast` may be a better method?

Reply:
Not unless you found seeing memory addresses particularly useful during development 🙂